tox.ini

```diff
@@ -53,33 +53,32 @@ deps =
     devdeps: git+https://github.com/astropy/astropy
     devdeps: git+https://github.com/matplotlib/matplotlib
 
+    # The 32bit and py39 build needs numpy installed before trying to install sunpy due
+    # to https://github.com/scikit-image/scikit-image/issues/4261
+    32bit,py39: numpy
+
     # Oldest deps we pin against.
-    oldestdeps: astropy<4.1
+    oldestdeps: astropy<4.2
     oldestdeps: numpy<1.17.0
     oldestdeps: matplotlib<3.2
-    oldestdeps: scipy<1.3
-    oldestdeps: parfive<1.2
+    oldestdeps: scipy<1.4
     oldestdeps: pandas<0.25
-    oldestdeps: parfive<1.2
-    oldestdeps: drms<0.7
     oldestdeps: h5netcdf<0.9
+    oldestdeps: parfive<1.3
+    oldestdeps: drms<0.7
+
     # These are specific online extras we use to run the online tests.
     online: pytest-rerunfailures
     online: pytest-timeout
     online: astroquery
     hypothesis: astroquery
 
-    # The 32bit and py39 build needs numpy installed before trying to install sunpy due
-    # to https://github.com/scikit-image/scikit-image/issues/4261
-    32bit,py39: numpy
     # We have to pin this to use a wheel for manylinux2010 to compile
     32bit: cryptography<3.0
 
     # Figure tests need a tightly controlled environment
-    figure-!devdeps: astropy==4.0.3
+    figure-!devdeps: astropy==4.2.0
     figure-!devdeps: matplotlib==3.3.2
-    # Temp measure until release
-    build_docs: git+https://github.com/sphinx-gallery/sphinx-gallery.git
```

Comment:
I don't think we can pull this yet, it doesn't look like there has been a release?

Reply:
Do we need it for the doc build?

Reply:
I guess not. :+1: